camel/messages/base.py

```diff
@@ -42,9 +42,20 @@ class BaseMessage:
     role_name: str
     role_type: RoleType
```

Comment:
These fieldnames are very confusing. I propose to introduce instead "society_role" for CAMEL-specific roles system and "llm_role" for openai roles.